modules/core/src/main/java/org/apache/ignite/internal/processors/cache/verify/VerifyBackupPartitionsTaskV2.java

```diff
@@ -186,7 +191,8 @@ public VerifyBackupPartitionsJobV2(VisorIdleVerifyTaskArg arg) {
                 }
 
                 if (!missingCaches.isEmpty()) {
-                    StringBuilder strBuilder = new StringBuilder("The following caches do not exist: ");
+                    StringBuilder strBuilder = new StringBuilder("The following caches do not exist or does not match to " +
```

Comment:
I think the given 'cacheKind' should be printed as well.
`The following caches do not exist or do not match to the given kind [kind=" + ... + ']: ');`

Reply:
Done.